File: amazia-core/src/main/java/com/example/salesreturn/controller/ListSalesReturnController.java
package com.example.salesreturn.controller;

import com.example.salesreturn.dto.AdminSalesReturnItem;
import com.example.salesreturn.service.ListSalesReturnService;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.List;

/**
 * 返品一覧 API（GET /api/sales-returns）。
 *
 * 想定呼び出し元: Amazia Console（JWT 検証経由）。
 *
 * 命名規約: docs/ai_context/operation_logs_naming.md
 *   action      : (取得系のため operation_logs には記録しない)
 *   screen_name : console.sales_returns.list
 *   api_name    : GET /api/sales-returns
 */
@RestController
@RequestMapping("/api")
public class ListSalesReturnController {

    private final ListSalesReturnService service;

    public ListSalesReturnController(ListSalesReturnService service) {
        this.service = service;
    }

    @GetMapping("/sales-returns")
    public ResponseEntity<List<AdminSalesReturnItem>> list() {
        return ResponseEntity.ok(service.list());
    }
}

File: amazia-console/app/SalesReturn/Controller/ListSalesReturnController.php
<?php

namespace App\SalesReturn\Controller;

use App\Http\Controllers\Controller;
use App\SalesReturn\Service\ListSalesReturnService;

/**
 * 返品申請一覧取得 Controller（GET /api/sales-returns）。
 *
 * Core の GET /api/sales-returns を中継し、Console フロントへ JSON を返す。
 * 認証は routes/api.php 側のミドルウェアで担保。
 */
class ListSalesReturnController extends Controller
{
    private ListSalesReturnService $service;

    public function __construct(ListSalesReturnService $service)
    {
        $this->service = $service;
    }

    public function __invoke()
    {
        $response = $this->service->list();
        return response()->json($response->json(), $response->status());
    }
}
